### ISSUE TITLE
Echo phenomena

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7306]
See 
https://www.ncbi.nlm.nih.gov/books/NBK565908/

**Comment by pnrobinson** [comment]
see also https://www.researchgate.net/publication/321117897_Pali_and_Echo_Phenomena_Symptoms_of_Persistence_and_Perseveration

**Comment by pnrobinson** [comment]
Label: Echophenomenon
Synoynm: Echo phenomenon, Echo phenomena
Proposed definition:
A stimulus triggered repetition of an action (echopraxia) or sound (echolalia) performed without the explicit awareness of an individual.
PMID: 25455694

-- current term Echolalia would be a child of this term. 
-- discuss which parent is most appropriate (Motor stereotypy?)

**Comment by pnrobinson** [comment]
@bcoleman96  -- is this still relevant in light of other new terms?

**Comment by pnrobinson** [comment]
@bcoleman96 coming back to this, let's try to finish v1 next week!

**Comment by bcoleman96** [comment]
I will forward the issue to Audrey and see what she thinks. This term would overlap with _Restricted or repetitive behaviors_ (HP:0031432). We could make it a child of that term and add a term for echopraxia. The only problem is that Echolalia is in a different part of the HPO under speech.

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000768
- label: Echophenomenon
- parents: [Motor stereotypy (HP:0000733)](https://hpo.jax.org/app/browse/term/HP:0000733)
- pmid: PMID:25455694
- def: A stimulus triggered repetition of an action (echopraxia) or sound (echolalia) performed without the explicit awareness of an individual.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7306
- synonyms:
